### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (#btn-login-submit)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#52]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[kh - Administration](http://localhost/kh/administrator/)
#### Element path
#btn-login-submit
#### Snippet
`<button class="btn btn-solid-primary btn-block btn-md" id="btn-login-submit">Log in</button>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.65 (foreground color: #ffffff, background color: #0184ff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 78.0.3904.97

**Comment by sahidhossen** [comment]
Highly accessible interfaces are easier on the eyes of the visually impaired, but harsher on the eyes of the normal visioned. The challenge is to meet the needs of both the majority and the minority. However, if you veer too far into one extreme, you’ll alienate a subset of your users. Most people don’t want to alienate the minority. But alienating the majority of your users is just as bad as alienating the minority. 

For this reason, the best design is a balanced one that satisfies the largest minority. We want to maintain a highly aesthetic design, that’s why we provide a local high contrast mode on the accessibility button. Aesthetics isn’t a subjective and trivial attribute used for ornamentation. It serves an important purpose in the user experience. It determines whether users trust your app, perceive it as valuable, or are satisfied using it. In other words, aesthetics affects user engagement and conversion rate. Discounting it is not only bad for users, but bad for application, but Accessibility extremists tend to discount aesthetics. 

They believe an interface should be as accessible as possible for the minority without considering how it affects the average user. The cross-section of the aesthetic-accessibility spectrum is the balance point for designing interfaces that satisfy the most users, so we avoid designing at the extreme ends of the spectrum and respect the aesthetic-accessibility paradox.

**Comment by micker** [comment]
note 1 : joomla is for all, we can't say this and ask to minority to actived something to be ok
Many gouvrenements wants (with law) acessibility site in both part (front and back)
its an immportant feature for all not just a for a minority

note 2 night theme isn't a good solution it just a cool feature initiate from amoled device to keep some watts .... 
i realy think that a good admin isn't only for designer eyes we need to find a good balance

=> Joomla! backend admin can be an example for all other ! not a design, a vision of web

**Comment by brianteeman** [comment]
Luckily I don't have to waste my time debunking your comments. It is a hard requirement of this template that it must be accessible.

**Comment by kawshar** [comment]
I have seen that all installation buttons have issues. Will be fixed ASAP. Btw, there is a new PR that fixed many issues. I hope someone with test and merge the PR.

Thanks

**Comment by brianteeman** [comment]
https://github.com/joomla-projects/j4adminui/pull/60#issuecomment-557029412
